lib/Pipeline.js

```diff
@@ -88,8 +89,14 @@ Pipeline.processJSONWithExtras  = function(gltfWithExtras, options) {
         removeNormals(gltfWithExtras);
     }
     addDefaults(gltfWithExtras, options);
+
+    // Generate stats for unoptimized input
+    if (options.stats) {
+        gltfStatistics.printStats(gltfWithExtras, options.inputPath);
+    }
```

Comment:
The reason I put this after `removeNormals` is because the gltfStatistics function requires the defaults to be set (`addDefaults`). Would it be ok to move the `addDefaults` above `removeNormals` too?